packages/plugin-core/src/_extension.ts

```diff
@@ -586,3 +605,37 @@ export function shouldDisplayLapsedUserMsg(): boolean {
     refreshMsg
   );
 }
+
+export function getVimExtensionResolvedKeybinding() {
```

Comment:
can we rename to make this a little clearer? something like `checkAndApplyVimKeybindingOverrideIfNotExist`(a mouthful but makes it really clear that we are doing something